views/dashboard/dashboard_help.html.twig

```diff
@@ -6,64 +6,52 @@
             </div>
             <div class="modal-body">
                 <h3>General Usage/ Plot Options</h3>
-                <p>To view any individual metric, select if from the dropdown in the <strong>Plot Options</strong> pane and
```

Comment:
I know this text isn't being changed here, but since I'm reviewing it, it looks like there's a typo here.  Was this supposed to read "select _it_ from the dropdown"?

Reply:
Likely. This is the area it's talking about:

![screen shot 2019-02-12 at 1 33 15 pm](https://user-images.githubusercontent.com/80459/52662827-ce3ebb80-2eca-11e9-8a91-ba6ca61d74fb.png)